tests/model/templates/sections/tester-response-section-parser.model.spec.ts

```diff
@@ -0,0 +1,113 @@
+import {
+  DisagreeCheckboxParser,
+  TesterResponseSectionParser
+} from '../../../../src/app/core/models/templates/sections/tester-response-section-parser.model';
+
+const EMPTY_DISAGREE_CHECKBOX = '- [ ] I disagree';
+const FILLED_DISAGREE_CHECKBOX = '- [x] I disagree';
+const DEFAULT_DISAGREEMENT_REASON = '[replace this with your reason]';
+const USER_DISAGREEMENT_REASON = 'I disagree!';
+
+const RESPONSE_TITLE = 'response';
+const RESPONSE_DESCRIPTION = 'Team chose [`response.Rejected`]';
+const RESPONSE_TEAM_CHOSE = 'Rejected';
+
+const SEVERITY_TITLE = 'severity';
+const SEVERITY_DESCRIPTION = 'Team chose [`severity.Low`]\nOriginally [`severity.High`]';
+const SEVERITY_TEAM_CHOSE = 'Low';
+const SEVERITY_TESTER_CHOSE = 'High';
+
+const TYPE_TITLE = 'type';
+const TYPE_DESCRIPTION = 'Team chose [`type.DocumentationBug`]\nOriginally [`type.FunctionalityBug`]';
+const TYPE_TEAM_CHOSE = 'DocumentationBug';
+const TYPE_TESTER_CHOSE = 'FunctionalityBug';
+
+const DUPLICATE_TITLE = 'duplicate status';
+const DUPLICATE_DESCRIPTION =
+  "Team chose to mark this issue as a duplicate of another issue (as explained in the _**Team's response**_ above)";
+
+export const RESPONSE_DISAGREEMENT =
+  '## :question: Issue response\n\n' +
+  'Team chose [`response.Rejected`]\n\n' +
+  '- [ ] I disagree\n\n' +
+  '**Reason for disagreement:** [replace this with your reason]\n\n' +
+  '-------------------';
```

Comment:
For more complicated formats, it would look better to use String.raw`text here`

Reply:
not sure if it this looks better

<img width="448" alt="Screenshot 2022-03-21 at 3 04 19 PM" src="https://user-images.githubusercontent.com/68138671/159217398-848df4a9-c5d8-4137-b2ef-b52d0da668c8.png">

Reply:
Hmm yea I didn't consider having to escape backticks. Although the overall format is much clearer (esp the spacing between each line.) The motivation is to be able to view a raw comment and be able to compare it to the test cases to see the difference quickly. 

See what the rest thinks @seanlowjk @gycgabriel @chunweii

Reply:
I personally think that the first version is better. It is not too complicated and there is no need to escape the backticks.